cmd/secrets-store-csi-driver/main.go

```diff
@@ -36,20 +36,16 @@ var (
 	endpoint   = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
 	driverName = flag.String("drivername", "secrets-store.csi.k8s.com", "name of the driver")
 	nodeID     = flag.String("nodeid", "", "node id")
+	volumePath = flag.String("volume", "", "Volume path for provider")
```

Comment:
Should we default this flag to `/etc/kubernetes/secrets-store-csi-providers` here?